src/routes/+layout.svelte

```diff
@@ -61,85 +78,103 @@
+      class="absolute z-50 font-bold p-10 mt-8 rounded-lg dark:bg-white bg-neutral-800 hover:underline underline-offset-4 text-white dark:text-black text-2xl left-1/2 transform -translate-x-1/2 -translate-y-[999%] focus:translate-y-1/2 transition-all duration-200 ease-in-out"
     >
-      Skip to content.
+      Skip to main content
     </a>
 
-    <a href="{base}/">
-      <picture class="block w-32 lg:w-48 self-center object-contain">
-        <source
-          type="image/avif"
-          srcset="{base}/images/knowledge-hub-logo.avif"
-        />
-        <source
-          type="image/webp"
-          srcset="{base}/images/knowledge-hub-logo.webp"
-        />
-        <img
-          class="w-full h-full object-contain"
-          height="180"
-          width="180"
-          src="{base}/images/knowledge-hub-logo.png"
-          alt="Climate Town Knowledge Hub"
-        />
-      </picture>
-    </a>
-    <DarkModeControl
-      cssClass="absolute w-fit top-1 right-1 border-2 border-current rounded-md p-2 bg-zinc-50 text-black dark:bg-zinc-900 dark:text-white"
-    ></DarkModeControl>
-
-    <nav class="gap-2 flex flex-wrap lg:flex-col">
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        href="{base}/">Home</a
-      >
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        href="{base}/resources">Resources</a
-      >
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        href="{base}/youtube">YouTube Feed</a
-      >
-      <div class="h-4" />
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        target="_blank"
-        rel="noreferrer"
-        href={github_url}
-        >✍ Contribute on GitHub <span class="sr-only">in a new tab</span></a
-      >
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        target="_blank"
-        rel="noreferrer"
-        href="{github_url}/#contributors"
-        >📣 Credits <span class="sr-only">in a new tab</span></a
-      >
-      <a
-        class="p-2 text-zinc-100 bg-green-700/75 dark:bg-green-950 font-bold rounded-lg"
-        target="_blank"
-        rel="noreferrer"
-        href={climate_town_url}
-        >🌐 Main Website <span class="sr-only">in a new tab</span>
-      </a>
-    </nav>
+    <div class="flex max-w-7xl mx-auto h-16 text-end">
+      <div class="mx-8 flex flex-row justify-between items-center w-full">
+        <a href="{base}/" class="flex flex-row items-center">
+          <h2 class="font-bold text-2xl">Knowledge Hub</h2>
+        </a>
+
+        <div
+          class="block md:hidden gap-2"
+          on:focusout={handleDropdownFocusLoss}
+        >
+          <DarkModeControl cssClass="w-fit p-2"></DarkModeControl>
+
+          <button on:click={handleDropdownClick} id="menu">
```

Comment:
one thing missing here, we need an accessible name for this menu button
<img width="340" alt="Screen Shot 2024-08-21 at 4 13 52 PM" src="https://github.com/user-attachments/assets/eed8ae35-6764-46a0-9444-a0eaa60dcf11">